### PR TITLE
moe: add page

### LINKED ISSUE #6347: page request: moe
https://www.gnu.org/software/moe/moe.html

### PR DESCRIPTION
**Pull request by pixelcmtd**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Closes #6347.

### CODE REVIEW

File: pages/common/moe.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# moe
+
+> A WYSIWYG text editor for ISO-8859-15 encoded text.
+> More information: <https://www.gnu.org/software/moe/moe.html>.
+
+- Edit a file:
+
+`moe {{path/to/file}}`
+
+- Save and Quit:
```
Comment (navarroaxel) [inline pages/common/moe.md:10]: Please move the hotkeys at the end, first the command examples.

File: pages/common/moe.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`Ctrl + X`
+
+- Open a file as read-only:
+
+`moe -o {{path/to/file}}`
+
+- Edit a file without creating backups, and ignoring case in searches:
+
+`moe -B -i {{path/to/file}}`
```
Comment (navarroaxel) [inline pages/common/moe.md:20]: I think we can split this example in 2.

File: pages/common/moe.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`moe {{path/to/file}}`
+
+- Save and Quit:
+
+`Ctrl + X`
+
+- Open a file as read-only:
+
+`moe -o {{path/to/file}}`
```
Comment (navarroaxel) [inline pages/common/moe.md:16]: We prefer long argument when are available:
```suggestion
`moe --read-only {{path/to/file}}`
```

File: pages/common/moe.md (hunk 1 above)
Comment (chrissxYT) [reply pages/common/moe.md:10]: is this mentioned in the content guidelines or style guide? or is that just another informal rule you have?

File: pages/common/moe.md (hunk 2 above)
Comment (chrissxYT) [reply pages/common/moe.md]: i had considered that and thought it would be better with both in one, but whatever

File: pages/common/moe.md (hunk 1 above)
Comment (chrissxYT) [reply pages/common/moe.md]: and also: shouldn't `tldr vim` then also be changed, because `:help` and `:wq` and listed before another command?

**Review by marchersimon — APPROVED** [review]
Looks good, thanks

**Review by blueskyson — APPROVED** [review]
LGTM.

File: pages/common/moe.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# moe
+
+> A WYSIWYG text editor for ISO-8859-15 encoded text.
+> More information: <https://www.gnu.org/software/moe/moe.html>.
+
+- Edit a file:
```
Comment (CleanMachine1) [inline pages/common/moe.md:6]:
```suggestion
- Edit a file and create a backup ({{insert what the format of the backup is here}} if edited :
```

Obviously tell me what the format is or just change it yourself in a commit.

File: pages/common/moe.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`moe {{path/to/file}}`
+
+- Open a file as read-only:
+
+`moe --read-only {{path/to/file}}`
+
+- Edit a file without creating backups:
+
+`moe -B {{path/to/file}}`
```
Comment (navarroaxel) [inline pages/common/moe.md:16]:
```suggestion
`moe --no-backup {{path/to/file}}`
```

File: pages/common/moe.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+- Edit a file ignoring case in searches:
+
+`moe -i {{path/to/file}}`
```
Comment (navarroaxel) [inline pages/common/moe.md:20]:
```suggestion
`moe --ignore-case {{path/to/file}}`
```

File: pages/common/moe.md (hunk 4 above)
Comment (marchersimon) [reply pages/common/moe.md:6]: I'd phrase it the same way as in `nano`: `Open nano and create a backup file (file~) when saving edits:`

Comment (navarroaxel) [reply pages/common/moe.md:6]:
```suggestion
- Open moe and create a backup file (file~) when saving edits:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**